src/reducers/contactsReducer.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash';
+import { FETCH_CONTACTS } from '../actions/types';
+
+export default (state = {}, action) => {
+  switch (action.type) {
+    case FETCH_CONTACTS:
+      return { ...state, ..._.mapKeys(action.payload, 'id') };
```

Comment:
if this is the only place you use lodash i would suggest using an es6 method instead of adding a new library just for this

Reply:
BIG LIKE! And remove lodash from package.json